DocumentedExamples/Zonally_Averaged_Global_Meridional_Overturning_Circulation.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    plt.title('Overturning in %s' % getvar_args['expt']);
</pre><p>String formatting here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>